### PR TITLE
kdd 2020 hands-on tutorial materials

### PR DESCRIPTION
**Pull request by iris-msds**

### CODE REVIEW

File: src/KDD2020-HandsOnTutorial/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+## KDD 2020 hands-on tutorial
+# In Search For A Cure: Recommendation with Knowledge Graph on CORD-19
```
Comment (DarrinEide) [inline src/KDD2020-HandsOnTutorial/README.md]: >#  [](start = 0, length = 2)

Single-hash header should always be the first content shown

File: src/MAG-Samples/KDD2020-HandsOnTutorial/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+## KDD 2020 hands-on tutorial
+# In Search For A Cure: Recommendation with Knowledge Graph on CORD-19
```
Comment (DarrinEide) [inline src/MAG-Samples/KDD2020-HandsOnTutorial/README.md]: >#  [](start = 0, length = 2)

Single has header should always be the first content shown

File: src/KDD2020-HandsOnTutorial/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+## KDD 2020 hands-on tutorial
```
Comment (DarrinEide) [inline src/KDD2020-HandsOnTutorial/README.md]: Should this page exist? You're linking to src/MAG-Samples/KDD2020-HandsOnTutorial from the main README, not this page

File: src/MAG-Samples/KDD2020-HandsOnTutorial/Module1-Get-MAG-SubGraph/README.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
+---
+title: MAG subgraph generation (PySpark)
+description: Using Azure Databricks PySpark to generate MAG subgraph by full MAG data
+language: Python
+products: Azure
+date: 08/10/2020
```
Comment (DarrinEide) [inline src/MAG-Samples/KDD2020-HandsOnTutorial/Module1-Get-MAG-SubGraph/README.md]: 8/25

File: src/MAG-Samples/KDD2020-HandsOnTutorial/README.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
+## KDD 2020 hands-on tutorial
```
Comment (DarrinEide) [inline src/MAG-Samples/KDD2020-HandsOnTutorial/README.md]: Needs header, similar to what you have in Module1 README

File: src/MAG-Samples/KDD2020-HandsOnTutorial/Module2-Cord19-SubGraph-Analytics/README.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
+# CORD-19 MAG Sub-graph Analysis (Pandas)
```
Comment (DarrinEide) [inline src/MAG-Samples/KDD2020-HandsOnTutorial/Module2-Cord19-SubGraph-Analytics/README.md:8]: Needs header, similar to what you have in Module1 README

**Review by DarrinEide — COMMENTED** [review]
Left a few minor comments

File: src/KDD2020-HandsOnTutorial/README.md (hunk 3 above)
Comment (iris-msds) [reply src/KDD2020-HandsOnTutorial/README.md]: deleted.

File: src/MAG-Samples/KDD2020-HandsOnTutorial/Module1-Get-MAG-SubGraph/README.md (hunk 4 above)
Comment (iris-msds) [reply src/MAG-Samples/KDD2020-HandsOnTutorial/Module1-Get-MAG-SubGraph/README.md]: updated

File: src/MAG-Samples/KDD2020-HandsOnTutorial/README.md (hunk 5 above)
Comment (iris-msds) [reply src/MAG-Samples/KDD2020-HandsOnTutorial/README.md]: added.

**Pull request merged**